package.json

```diff
@@ -19,15 +25,15 @@
   "license": "ISC",
   "dependencies": {
     "@braintree/sanitize-url": "^7.0.1",
+    "https-proxy-agent": "^7.0.4",
```

Comment:
not sure we need this one in deps tbh

Reply:
The only real dep is `mockttp`, the others are just used in my demo proxy.config.ts